Source/Urho3D/Physics/RaycastVehicle.cpp

```diff
@@ -330,7 +330,7 @@ void RaycastVehicle::FixedPostUpdate(float timeStep)
             }
             if (skidInfoCumulative_[i] > 0.05f)
             {
-                skidInfoCumulative_[i] -= 0.002;
+                skidInfoCumulative_[i] -= 0.002f;
```

Comment:
BTW, what's this?

Reply:
fixed warning double to float runcation

Reply:
Damn, sorry. I thought you've changed the value.